src/Stratis.Bitcoin.IntegrationTests/RPC/RPCSettingsTest.cs

```diff
@@ -1,44 +0,0 @@
-﻿using NBitcoin;
```

Comment:
Shall we keep this test but use the settings init with RPCsettings?

Reply:
This test is basically testing if callback was called. If we keep it but without a callback it won't make much sense imho

Reply:
ok